VehicleInformation/GPSData.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+
+namespace AutonomousCars
+{
+    public class GPSData
+    {
+        // Latitude value in degrees [-90.000000:90.000000]
+        // ( 90 degrees south to 90 degrees north )
```

Comment:
Nice approach on documenting the fields. 
There is a standard to add documentation and the IDE can pick it up and display it when hovering / inspecting a method: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/examples

Similar in other languages too.